Utils/fetchUtils.js

```diff
@@ -0,0 +1,15 @@
+import axios from 'axios'
+
+const NCnewsAPI = axios.create({
+  baseURL: "https://nc-news-tm65.onrender.com",
+});
+
+export const fetchArticles = () => {
+  return NCnewsAPI.get(`/api/articles`)
+      .then((res) => {
+      return res.data;
+    })
+    .catch((err) => {
```

Comment:
There should be no catch blocks in your api calls, your back end will handle this